cmd/expiration-mailer/main.go

```diff
@@ -152,13 +157,14 @@ func (m *mailer) findExpiringCertificates() error {
 	now := m.clk.Now()
 	// E.g. m.NagTimes = [1, 3, 7, 14] days from expiration
 	for i, expiresIn := range m.nagTimes {
+		expiresIn += nagMargin
```

Comment:
`expiresIn` is dependent on `m.clk.Now()` on L58 so it has to be done here I think.